ape_safe/accounts.py

```diff
@@ -103,17 +176,19 @@ def fallback_handler(self) -> Optional[ContractInstance]:
         slot = keccak(text="fallback_manager.handler.address")
         value = self.provider.get_storage_at(self.address, slot)
         address = self.network_manager.ecosystem.decode_address(value[-20:])
-        if address != ZERO_ADDRESS:
-            return self.chain_manager.contracts.instance_at(address)
-        else:
-            return None
+        return (
+            self.chain_manager.contracts.instance_at(address) if address != ZERO_ADDRESS else None
+        )
 
     @cached_property
-    def client(self) -> SafeClient:
+    def client(self) -> BaseSafeClient:
         chain_id = self.provider.chain_id
         if chain_id not in self.account_file["deployed_chain_ids"]:
```

Comment:
**thought (NON-BLOCKER**: For forked networks that don't have known safe addresses, we could treat it like local. That probably is pretty rare if non-existent use-case, but I don't see why not.

Reply:
fixed, but requires [another ape core pr to ](https://github.com/ApeWorX/ape/pull/1714)